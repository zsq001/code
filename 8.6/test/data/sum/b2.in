100 100
232116 1371141 2446800 128026 829394 1273786 19782 2418473 224728 2776293 1638814 2882350 2264441 974097 3039259 1624787 737007 915115 2439884 2626792 2170510 614076 371989 1660540 2629019 449985 3118122 1046054 1244887 3161766 300225 1474080 1107496 1365298 1880467 1336936 456179 1215741 389398 2453502 950257 888397 2004566 364299 3129485 2060334 2712555 779510 2175011 2160898 783977 641865 646952 259202 2893653 2420476 1894044 3183290 1508965 2423138 615975 2645321 1727911 2509683 3215625 2045826 2411098 2574608 2871026 848817 975224 143766 682026 2421260 1795939 3149075 1295978 2774722 1938845 762695 2048357 2312832 356879 264613 2489976 1985778 996549 1620860 876226 2824107 3030933 2639266 1375457 2691430 1516303 458900 1490271 249498 1455564 776944 
1 23 95 2074287
2 4 57
1 25 70 1271849
2 68 99
1 56 86 1950655
2 46 57
1 43 82 2040231
2 65 88
1 18 41 2246145
2 56 92
1 16 26 382596
2 61 75
1 26 100 2122555
2 28 39
1 44 58 3081466
2 39 46
1 61 67 3182143
2 33 80
1 69 78 2251783
2 76 98
1 19 34 1055559
2 23 76
1 21 82 1858500
2 1 36
1 22 85 2964580
2 38 44
1 4 67 2732004
2 5 89
1 47 72 1616349
2 33 47
1 37 50 1359080
2 30 76
1 47 51 1348310
2 71 86
1 2 74 630766
2 23 73
1 19 57 178308
2 37 97
1 24 41 3282643
2 3 73
1 76 97 870231
2 30 73
1 62 86 2364642
2 58 74
1 19 55 872929
2 23 65
1 27 54 1440110
2 21 31
1 26 94 1246129
2 37 75
1 1 91 763362
2 42 42
1 51 95 551621
2 6 98
1 53 55 1503947
2 47 58
1 48 49 1176788
2 29 44
1 47 83 113357
2 18 87
1 36 40 424064
2 95 95
1 13 85 2292286
2 4 84
1 46 63 256593
2 54 63
1 15 32 2343048
2 28 86
1 28 37 1775613
2 40 67
1 11 79 3238882
2 18 20
1 69 98 1541500
2 59 79
1 39 71 159142
2 31 87
1 47 97 2082718
2 31 40
1 33 99 2790420
2 40 58
1 35 57 1178129
2 30 88
1 19 75 3281709
2 68 97
1 38 93 1354748
2 28 60
1 58 86 248465
2 27 40
1 36 81 1531024
2 32 75
1 23 24 2949846
2 21 27
1 2 51 2189557
2 13 65
1 7 55 1351458
2 46 48
1 36 69 1868215
2 40 53
1 51 91 2392211
2 1 28
