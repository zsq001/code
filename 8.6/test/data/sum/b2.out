804239812
651896909
421826923
733757988
852310057
829849914
869304126
182460238
480185417
242697909
455430151
877932576
865207872
273830525
740603612
109803059
138424042
565007251
377940600
18866465
583854224
515893960
845391401
804873035
820959869
934953210
137474259
408004576
136756754
514611403
673479472
317825282
637907257
617107208
789621831
866096107
696904270
310739602
340807844
268014940
830244430
847016202
272212745
319021670
96179322
843436495
736366326
271978583
943874804
719159719
